ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt

```diff
@@ -273,7 +274,7 @@ class GodToolsShortcutManager @Inject constructor(
             Tool.Type.TRACT -> context.createTractActivityIntent(code, *locales.toTypedArray())
             Tool.Type.ARTICLE -> context.createCategoriesIntent(code, locales[0])
             else -> return@withContext null
-        }.apply { action = Intent.ACTION_VIEW }
+        }.apply { action = Intent.ACTION_VIEW; putExtra(SHORTCUT_LAUNCH, true) }
```

Comment:
putting multiple statements on a single line is something that in general is a Java/Kotlin anti-pattern.

Taking a second look at the surrounding logic, let's move the adjustments to the intent out of the apply block and just execute them as normal statements. e.g. `intent.action = Intent.ACTION_VIEW`

Reply:
@frett is this the correct way to do it? 
<img width="651" alt="Screen Shot 2020-10-26 at 4 25 41 PM" src="https://user-images.githubusercontent.com/39680460/97224775-02920280-17a8-11eb-96a7-eca0bf57a510.png">

Reply:
yup, I would get rid of the extra line return between creating the intent and updating the action/extras. 

The rationale being that all 3 of the statements deal with initializing the intent, so they can be kept together so that it's is easier to scan over the code and see a block of code that creates the appropriate intent.